### PR TITLE
DTSAM-524: Adding FTA for IBCA

### PR DESCRIPTION
**Pull request by SivaKanukolluQA**
### Jira link
https://tools.hmcts.net/jira/browse/DTSAM-524

### Change description
Added S-162 FTA for IBCA role

> This PR is branched from #2084

### Checklist
- [ ] Does this PR introduce a breaking change - No

### CODE REVIEW

**Review by abul-syed — DISMISSED** [review]
lgtm

**Review by mikebrownccd — DISMISSED** [review]
LGTM

File: charts/am-org-role-mapping-service/values.preview.template.yaml [hunk 1]
```diff
@@ -82,7 +82,8 @@ orm:
 
 am-role-assignment-service:
   java:
-    image: 'hmctspublic.azurecr.io/am/role-assignment-service:latest'
+    # temporary override until hmcts/am-role-assignment-service#2329 is merged: image: 'hmctspublic.azurecr.io/am/role-assignment-service:pr-2329'
```
Comment (mattnayler) [inline charts/am-org-role-mapping-service/values.preview.template.yaml:85]: Temporary override must be removed before merging

Comment (mattnayler) [reply charts/am-org-role-mapping-service/values.preview.template.yaml]: This change temporary override has now been removed.

**Review by mattnayler — APPROVED** [review]

**Review by abul-syed — APPROVED** [review]
lgtm

**Pull request merged**